### ISSUE TITLE
Populate the README with a description of the project

### ISSUE THREAD

**Issue by ryan-whit** [johnaslani/staffing#12]
One of the requirements from the project is that it has a quality README file. Our current file is mostly empty, so it needs to be populated.

From the requirements:

```
Have a quality README (with unique name, description, technologies used, screenshot, and link to deployed application).
```

## Exit criterion

These tasks must be added:

- [x] Unique name
- [x] project description
- [x] list of technologies used, with a brief description
- [x] screenshot of the web page
- [x] link to the deployed application
- [x] link to the Git repo

**Comment by johnaslani** [comment]
I work on README file.

**Comment by ryan-whit** [comment]
Closed by #26
